astropy/coordinates/tests/test_funcs.py

```diff
@@ -38,8 +38,10 @@ def test_constellations():
     from ..funcs import get_constellation
 
     inuma = ICRS(9*u.hour, 65*u.deg)
-    res = get_constellation(inuma)
-    res_short = get_constellation(inuma, short_name=True)
+    with pytest.warns(ErfaWarning, match='ERFA function "utctai" yielded '
+                      '1 of "dubious year (Note 3)"'):
+        res = get_constellation(inuma)
+        res_short = get_constellation(inuma, short_name=True)
```

Comment:
I wonder whether we should solve the underlying issue here - it doesn't seem like users should get a warning here?

Reply:
Hopefully @eteq or @adrn can comment on this. This warning has been there for who-knows-how-long but it is currently ignored in the CI as per `pytest` settings.

Reply:
I think there is an argument that this is indeed a "correct" warning.  It stems from the fact that constellation boundaries were defined based on boundaries defined in 1875.  And ERFA considers 1875 to be a "dubious year" because some of the precession models aren't really meant to be used that far back. It's not a big enough affect to matter much for constellations, though.

Reply:
#7999 has now been merged, fixing this!

Reply:
Thx, I'll rebase